src/porting.h

```diff
@@ -220,6 +224,20 @@ inline u64 getDeltaMs(u64 old_time_ms, u64 new_time_ms)
 	return (old_time_ms - new_time_ms);
 }
 
+inline void preciseSleepUs(u64 sleep_time)
+{
+	if (sleep_time > 0)
+	{
+		u64 time = porting::getTimeUs();
+		u64 target_time = time + sleep_time;
+		if (sleep_time > SLEEP_ACCURACY)
+			sleep_us(sleep_time-SLEEP_ACCURACY);
+
+		// Busy-wait the remaining time to adjust for sleep inaccuracies
+		while ((s64)(target_time - porting::getTimeUs()) > 0);
```

Comment:
```suggestion
		u64 target_time = porting::getTimeUs() + sleep_time;
		if (sleep_time > SLEEP_ACCURACY)
			sleep_us(sleep_time - SLEEP_ACCURACY);

		// Busy-wait the remaining time to adjust for sleep inaccuracies
		// The target - now > 0 construct will handle overflow gracefully (even though it should never happen)
		while (target_time - porting::getTimeUs() > 0) {}
```

Reply:
I applied the patch manually, without removing `s64` as explained above, and also splitting the comment so it doesn't exceed 95 characters on one line.